bin/dustc

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env node
 
 var cli = require('cli').enable('glob', 'version'),
-    dust = require('../lib/server.js'),
+    dust = require('../index.js'),
```

Comment:
being extra snarky just because this is purely a convention fix and IMO convention is to leave out the extensions.